tick/u.q

```diff
@@ -3,16 +3,16 @@
 /2006.05.08 add
 
 \d .u
-init:{w::t!(count t::tables`.)#()}
+init:{(w;t)::(t!();tables`.)}
 
-del:{w[x]_:w[x;;0]?y};.z.pc:{del[;x]each t};
+(del;.z.pc):({w[x]_:w[x;;0]?y};{del[;x]each t})
 
-sel:{$[`~y;x;select from x where sym in y]}
+sel:{(select from x where sym in y;x)`~y}
```

Comment:
Well, the main reason for this change was because it's shorter :)
I still believe it's quite clear, so I'll keep it until our master @neutropolis makes the decision

Reply:
Then, just to drag my point across, should we change all `if`s to this syntax?
```q
(then;)cond;
```
I.e:
```q
if[x~`;:sub[;y]each t];
// to
(:sub[;y]each t];)x~`;
```

Haven't tested it yet, but I'll do so whenever I get to download v4.1

Reply:
This is not new in 4.1. We are simply indexing. For example, in this case, x~ will return 0b or 1b. The idea is to index and return the first or second position from the list on the right. I believe this is suitable for small and simple cases.

Reply:
My point still stands. I personally would keep it as it was, but if we are changing it then we should keep it consistent and change every instance of a conditional statement OR every if/else (`$`) statement. Let's wait to see what @neutropolis has to say about this.